docs/application/native/guides/multimedia/overview.md

```diff
@@ -89,6 +89,10 @@ You can use the following multimedia features in your native applications:
 
   You can use the camera to preview and capture images. You can capture still images with the device's internal camera and keep images on your target device.
 
+- [WebRTC](webrtc.md)
+
+  You can have real-time audio/video communication with a remote peer. You can also send and receive generic data with a remote peer.
+
 ## Related Information
```

Comment:
For second headings, please use sentence style capitalization
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Related information
```